app/views/classrooms/_classroom_students_table.html.erb

```diff
@@ -48,6 +51,15 @@
                 $0.00
               <% end %>
             </td>
+            <% if @can_manage_students %>
+              <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-900 font-medium">
+                <% if student.portfolio %>
```

Comment:
It looks a little too defensive to me. 🤔 

Since it's a student, we can assume a portfolio already exists. 

https://github.com/rubyforgood/stocks-in-the-future/blob/main/app/models/student.rb#L18-L20

What do you think? We can keep it if it's safer.